### DISCUSSION TITLE
How to speed up network speed using onedrive and raidrive

### DISCUSSION THREAD

**Discussion by jonny77** [Chia-Network/chia-blockchain#5642] [Ideas]
I upload   plots files  to onedrive and use raidrive to mount as local disk . but farming time is more than 5 seconds.   how to speed up this

Looking up qualities on \\RaiDrive-123\OneDrive China\home\plot-k32-2021-05-10-20-27-f501702.plot took: 21.30944013595581. This should be below 5 seconds to minimize risk of losing rewards.
2021-05-21T05:54:33.569 harvester chia.harvester.harvester: INFO     1 plots were eligible for farming 28cdf3b22a... Found 0 proofs. Time: 21.31236 s. Total 16 plots

**Comment by SackofFluor** [comment]
just a quick question...
you store your plots on a CLOUD service? Correkt?

When the harvesters checks the plot, what do you think is going to happen?
Your request has to travel to the cloud, be processed and then return to you....

What do you think could therefore be the most beneficial thing you could do?
Just a fruit for thoughts...
